src/main/java/com/google/cloud/spanner/pgadapter/statements/PgCatalog.java

```diff
@@ -384,7 +384,6 @@ public class PgType implements PgCatalogTable {
             + "  || (case typname\n"
             + "        when 'int2' then 'false'\n"
             + "        when 'int4' then 'false'\n"
-            + "        when 'float4' then 'false'\n"
```

Comment:
This is the 'typisdefined' column in `pg_type`. Up until now we listed `float4` as a 'type that we know, but that is not implemented'.